src/lib/useCalculateKclExpression.ts

```diff
@@ -72,11 +74,12 @@ export function useCalculateKclExpression({
   }, [programMemory, newVariableName])
 
   useEffect(() => {
-    if (!programMemory || !selectionRange) return
+    if (!programMemory) return
     const varInfo = findAllPreviousVariables(
       kclManager.ast,
       kclManager.programMemory,
-      selectionRange
+      // If there is no selection, use the end of the code
+      selectionRange || [code.length, code.length]
```

Comment:
This hook wouldn't work if nothing was selected within the code, which is true of operations like offset plane. I still can't get the autocompletions to work in the KCL argument input box though.